README.md

```diff
@@ -30,6 +30,13 @@ EXAMPLES
   $ yarn outdated --json | $(yarn bin)/format-yarn-outdated --excludes /path/to/excludes.yml --changelogs /path/to/changelogs.yml
   $ yarn outdated --json | $(yarn bin)/format-yarn-outdated --format json | jq '.minor[],.patch[] | .[0]' | xargs -I{} yarn upgrade {}
   $ yarn outdated --json | $(yarn bin)/format-yarn-outdated --format mackerel | mkr throw --service ServiceMetricName
+
+NPM SUPPORT
```

Comment:
Please copy the paragraph to `bin/format-yarn-outdated.js`

https://github.com/masawada/yarn-outdated-formatter/blob/8a58af4e29510ea667cc23472d8d025e8a660b4c/bin/format-yarn-outdated.js#L11-L29